### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by luisarmandodv**
@albertm805 @JuanCrg90 

Link: https://github.com/magma-labs/shopify-themes/issues/21

Original Flats:
36288998-237ff452-1283-11e8-9481-0439088cfa26
captura de pantalla 2018-02-15 a la s 12 51 41
captura de pantalla 2018-02-15 a la s 12 52 39

How it look:
captura de pantalla 2018-02-23 a la s 10 08 20
captura de pantalla 2018-02-23 a la s 10 08 43
captura de pantalla 2018-02-23 a la s 10 08 51

### CODE REVIEW

File: MH-FiorellaTheme5/source/javascripts/site.js [hunk 1]
```diff
@@ -1 +1,12 @@
 // This is where it all goes :)
+function openNav() {
+    document.getElementById("mySidenav").style.width = "95%";
+    document.getElementById("logo").src = "../images/home/logo_fiorella_menu.png";
+    document.getElementById("logo").style.zIndex = "10";
+}
+
+function closeNav() {
```
Comment (magmahound) [inline MH-FiorellaTheme5/source/javascripts/site.js]: 'closeNav' is defined but never used.

File: MH-FiorellaTheme5/source/javascripts/site.js [hunk 2]
```diff
@@ -1 +1,12 @@
 // This is where it all goes :)
+function openNav() {
```
Comment (magmahound) [inline MH-FiorellaTheme5/source/javascripts/site.js]: 'openNav' is defined but never used.

File: MH-FiorellaTheme5/source/javascripts/site.js [hunk 3]
```diff
@@ -1 +1,12 @@
...
+    document.getElementById("mySidenav").style.width = "95%";
+    document.getElementById("logo").src = "../images/home/logo_fiorella_menu.png";
+    document.getElementById("logo").style.zIndex = "10";
+}
+
+function closeNav() {
+    document.getElementById("mySidenav").style.width = "0";
+    document.getElementById("logo").src = "../images/home/wlogo_fiorella_mobil.png";
```
Comment (magmahound) [inline MH-FiorellaTheme5/source/javascripts/site.js]: Line is too long.

File: MH-FiorellaTheme5/source/javascripts/site.js [hunk 4]
```diff
@@ -1 +1,12 @@
 // This is where it all goes :)
+function openNav() {
+    document.getElementById("mySidenav").style.width = "95%";
+    document.getElementById("logo").src = "../images/home/logo_fiorella_menu.png";
```
Comment (magmahound) [inline MH-FiorellaTheme5/source/javascripts/site.js]: Line is too long.

File: MH-FiorellaTheme5/source/index.html.erb [hunk 5]
```diff
@@ -1,17 +1,13 @@
...
-  <path class="middleman-logo__bottom-left-bar" fill-opacity=".45" d="M0 300l200 40v-30L0 280z"/>
-  <path class="middleman-logo__bottom-right-bar" fill="#fff" d="M200 340l200-40v-20l-200 30z"/>
-</svg>
+<div class="container">
+  <%= partial "partials/_slide-out-menu" %>
 
-<h1>
-  Middleman is Running
-</h1>
+  <%= partial "partials/_header" %>
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/index.html.erb]: You don't need to add the `_` before the partial name. The partial gets detected just by using the name, no need to use the underscore. Can you please remove all of them from the partial rendering. 👍

File: MH-FiorellaTheme5/source/javascripts/site.js (hunk 4 above)
Comment (albertm805) [inline MH-FiorellaTheme5/source/javascripts/site.js]: Why are you using javascript to set the src logo image? Please remove this as it is a bad practice. 
Or give me an explanation on why you are using this?

File: MH-FiorellaTheme5/source/javascripts/site.js [hunk 6]
```diff
@@ -1 +1,12 @@
...
+    document.getElementById("logo").style.zIndex = "10";
+}
+
+function closeNav() {
+    document.getElementById("mySidenav").style.width = "0";
+    document.getElementById("logo").src = "../images/home/wlogo_fiorella_mobil.png";
+    document.getElementById("logo").style.zIndex = "2";
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/javascripts/site.js]: Also this, please remove it. All styles should be set with css.

File: MH-FiorellaTheme5/source/layouts/layout.erb [hunk 7]
```diff
@@ -3,51 +3,13 @@
...
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <!-- Use the title from a page's frontmatter if it has one -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="https://use.fontawesome.com/releases/v5.0.6/css/all.css" rel="stylesheet">
     <title><%= current_page.data.title || "Middleman" %></title>
     <%= stylesheet_link_tag "styles" %>
+    <%= javascript_include_tag "site" %>
   </head>
   <body>
-    <div class="container">
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/layouts/layout.erb]: 👍

File: MH-FiorellaTheme5/source/partials/_body-message.erb [hunk 8]
```diff
@@ -0,0 +1,14 @@
+<section class="body-message">
+  <div class="title-message">
+    <h2 class="title">
+        <span>ALWAYS FRESH</span>
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/partials/_body-message.erb]: Why are you inserting a span inside a header?

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 9]
```diff
@@ -1,116 +1,399 @@
...
   padding: 40px 20px;
+  text-align: center;
 }
+
 h2 {
   letter-spacing: 3px;
-  position: relative;
-  margin-top: 20px;
+	margin-top: 20px;
+	position: relative;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Your spacing here is off, you have double spacing.

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 10]
```diff
@@ -1,116 +1,399 @@
...
-  display: block;
-  border-top: solid 1px #E83671;
-  width: 100%;
-  height: 1px;
-  position: absolute;
-  top: 50%;
-  z-index: 1;
+	border-top: solid 1px #E83671;
+	content: "";
+	display: block;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Also here you have double spacing. Weird, can you change it please

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 11]
```diff
@@ -1,116 +1,399 @@
...
+	height: 1px;
+	position: absolute;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 🤔 position absolute what for? and why do you need the `:before`?

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 12]
```diff
@@ -1,116 +1,399 @@
...
+	z-index: 1;
 }
 
 h2.title span {
   background: #F3F9EF;
-  padding: 0 10px;
-  position: relative;
-  z-index: 5;
+	padding: 0 10px;
+	position: relative;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Again, weird indentation.

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 13]
```diff
@@ -1,116 +1,399 @@
...
+	z-index: 5;
 }
+
 .text-message{
   font-family: "MerriweatherSans-Light";
-  line-height: 2em;
   font-size: 22px;
-
+  line-height: 2em;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Keep it consistent. You either use `em` or `rem` or `px`. Looking at the rest of the code you are using `px` so I would stick to using that.

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 14]
```diff
@@ -1,116 +1,399 @@
...
+    width: 25px;
+  }
+    /*</SearchBar> */
+
+  /*<homeMenu>*/
+  .sidenav {
+    background-color: white;
+		height: 100%;
+		left: 0;
+		overflow-x: hidden;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Weird spacing again. 🤔

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 15]
```diff
@@ -1,116 +1,399 @@
...
+    align-items: center;
+    margin-top: 30px;
+    padding: auto;
+    opacity: 1;
+  }
+
+  .header {
+    background-image: url("../images/home/header_background.jpg");
+  	background-repeat: no-repeat;
+  	background-size: 100%;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Spacing.

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 16]
```diff
@@ -1,116 +1,399 @@
...
+  	height: 700px;
+  	margin:0;
+  	width: auto;
+  }
+
+  .upper-row {
+    background-color: #F3F9EF;
+  	height: 300px;
+  	opacity: 0.85;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Here also, spacing.

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 17]
```diff
@@ -1,116 +1,399 @@
...
+
+  .mobile-search-container {
+    display: none;
+  }
+
+  .topnav {
+    background-color: #F3F9EF;
+		border-bottom: 1px solid red;
+		margin-left: 40px;
+		margin-right: 40px;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 🤔 spacing.

File: MH-FiorellaTheme5/source/javascripts/site.js [hunk 18]
```diff
@@ -1 +1,12 @@
 // This is where it all goes :)
+function openNav() {
+    document.getElementById("mySidenav").style.width = "95%";
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/javascripts/site.js]: the JS style recommends to use single quotes instead double quotes.

File: MH-FiorellaTheme5/source/javascripts/site.js (hunk 4 above)
Comment (JuanCrg90) [reply MH-FiorellaTheme5/source/javascripts/site.js]: the JS style recommends to use single quotes instead double quotes.

File: MH-FiorellaTheme5/source/javascripts/site.js [hunk 19]
```diff
@@ -1 +1,12 @@
...
+    document.getElementById("logo").src = "../images/home/logo_fiorella_menu.png";
+    document.getElementById("logo").style.zIndex = "10";
+}
+
+function closeNav() {
+    document.getElementById("mySidenav").style.width = "0";
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/javascripts/site.js]: the JS style recommends to use single quotes instead double quotes.

File: MH-FiorellaTheme5/source/stylesheets/styles.css (hunk 11 above)
Comment (luisarmandodv) [reply MH-FiorellaTheme5/source/stylesheets/styles.css]: @IsmaelVillavicencio  is going to fix this.

File: MH-FiorellaTheme5/source/javascripts/site.js [hunk 20]
```diff
@@ -1 +1,12 @@
 // This is where it all goes :)
+function openNav() {
+  document.getElementById('logo').className = 'pinkLogo';
+  document.getElementById('mySidenav').classList.remove('closeSideNav');
+  document.getElementById('mySidenav').classList.add('openSideNav');
+}
+
+function closeNav() {
+  document.getElementById("logo").className = 'whiteLogo';
```
Comment (magmahound) [inline MH-FiorellaTheme5/source/javascripts/site.js]: Mixed double and single quotes.

File: MH-FiorellaTheme5/source/partials/_body-message.erb (hunk 8 above)
Comment (luisarmandodv) [reply MH-FiorellaTheme5/source/partials/_body-message.erb]: @IsmaelVillavicencio did this correction on his branch.... Do I need to fix it?

**Pull request closed without merging**